app/services/enqueue_index_jobs.rb

```diff
@@ -0,0 +1,70 @@
+# Indexes a book version thru the SQS queuing system
+#
+# @params indexing_version e.g., Search::BookVersions::I1::IndexingStrategy::VERSION
+class EnqueueIndexJobs
+  def initialize
+    @todo_jobs_queue = TodoJobsQueue.new
+  end
+
+  def process_rex_releases
+    rex_releases = OpenStax::RexReleases.new
+    books_ids = rex_releases.map{ |release| release.books }.flatten
+
+    books_ids.each do |book_id|
+      ACTIVE_INDEXING_STRATEGIES.each do |indexing_strategy|
+        process_book(book_id, indexing_strategy.new.version)
+      end
+    end
+
+    check_for_books_to_delete
```

Comment:
I understand the desire to put code into private methods, but here it doesn't help me read this code and understand what's going on.  "process_book" is kind of vague.  Also, `process_book` sets state that is used in `check_for_books_to_delete` so for that I'd rather just have the code more inline here.  I made a PR into this PR to show you what I mean - it ends up as 24 lines of code but keeps logic together and is more readable to me (note that that PR is pseudocode-y so you may not just be able to merge it).